tailwind.config.cjs

```diff
@@ -1,6 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./src/**/*.{html,js,ts}', './index.html'],
+  content: ['./src/client/**/*.{html,js,ts}', './index.html'],
```

Comment:
This misses the `src/server` folder, why skip it?

Reply:
Won't the client folder hold all Tailwind CSS classes?